inc/utils/namespace.php

```diff
@@ -1094,14 +1094,13 @@ function get_s3_client( array $args = [] ) : ? S3Client {
 /**
  * Make an HTTP request to ClickHouse.
  *
- * Returns null for zero results, object for a single row / or an array of objects.
- *
  * @param string $query SQL statement, if $body is present it will be encoded into the request URL.
- * @param string $body Optional query body. For use with queries like INSERT with JsonEachRow format.
- * @param string $return Optional return type. Can be 'auto', 'array', 'object' or 'raw'.
+ * @param array $params Array of query parameters to add to the query string, use `param_` prefixed keys for interpolation of values.
+ * @param string $return Optional return type. Can be 'array', 'object' or 'raw', default 'array'.
+ * @param string|null $body Optional query body. For use with queries like INSERT with JsonEachRow format.
  * @return null|\stdClass|\stdClass[]|WP_Error
  */
-function clickhouse_query( string $query, string $body = '', string $return = 'auto' ) {
+function clickhouse_query( string $query, array $params = [], string $return = 'array', ?string $body = null ) {
```

Comment:
I know the function's already here and named, but starting with a verb would be preferential imo (i.e. `query_clickhouse()`)

Reply:
Realised the ES stuff was still in there so removed it and renamed back to just `Utils\query()`